tests/Fixer/Alias/SetTypeToCastFixerTest.php

```diff
@@ -0,0 +1,161 @@
+    {
+        return [
+            [
+                '<?php (array)    $foo;',
+                '<?php settype  (  $foo  , \'array\');',
+            ],
+            [
+                '<?php (bool)    $foo;',
+                '<?php settype  (  $foo  , "Bool");',
+            ],
+            [
+                '<?php (bool)    $foo;',
+                '<?php settype  (  $foo  , "boolean");',
+            ],
+            [
+                '<?php (float) $foo;',
+                '<?php settype($foo, "double");',
+            ],
+            [
+                '<?php (float) $foo;',
+                '<?php settype($foo, "float");',
+            ],
+            'int full caps' => [
+                '<?php (int) $foo;',
+                '<?php settype($foo, "INT");',
+            ],
+            [
+                '<?php (int) $foo;',
+                '<?php settype($foo, "integer");',
+            ],
+            [
+                '<?php echo 1; (object) $foo;',
+                '<?php echo 1; settype($foo, "object");',
+            ],
+            [
+                '<?php (string) $foo;',
+                '<?php settype($foo, "string");',
+            ],
+            'no space' => [
+                '<?php (int) $foo;',
+                '<?php settype($foo,"integer");',
+            ],
+            [
+                '<?php (int) $foo;',
+                '<?php settype($foo, "integer");',
+            ],
+            'no space comments' => [
+                '<?php /*0*//*1*/(int)/*2*//*3*/$foo/*4*//*5*//*6*//*7*/;/*8*/',
+                '<?php /*0*//*1*/settype/*2*/(/*3*/$foo/*4*/,/*5*/"integer"/*6*/)/*7*/;/*8*/',
+            ],
+            'comments with line breaks' => [
+                '<?php #0
+#1
+(int)#2
+#3
+#4
+$foo#5
+#6
+#7
+#8
+;#9',
+                '<?php #0
+#1
+settype#2
+#3
+(#4
+$foo#5
+,#6
+"integer"#7
+)#8
+;#9',
+            ],
+            // do not fix cases
+            'first argument is not a variable' => [
+                '<?php settype(1, "double");',
+            ],
+            'first argument is variable followed by operation' => [
+                '<?php settype($foo + 1, "integer"); // this is not valid code, so do not fix it',
+            ],
+            'wrong numbers of arguments' => [
+                '<?php settype($foo, "integer", $a);',
+            ],
+            'other namespace I' => [
+                '<?php a\b\settype($foo, "integer", $a);',
+            ],
+            'other namespace II' => [
+                '<?php \b\settype($foo, "integer", $a);',
+            ],
+            'static call' => [
+                '<?php A::settype($foo, "integer");',
+            ],
+            'member call' => [
+                '<?php $a->settype($foo, "integer");',
+            ],
+            'unknown type' => [
+                '<?php $a->settype($foo, "foo");',
+            ],
+            'no null cast' => [
+                '<?php $a->settype($foo, "null");',
```

Comment:
the above two are testing on member call when they shouldn't be

Reply:
copy-paste error :(  good catch!